prowler/providers/aws/services/rds/rds_instance_no_public_access/rds_instance_no_public_access.py

```diff
@@ -14,14 +15,22 @@ def execute(self):
             if not db_instance.public:
                 report.status = "PASS"
                 report.status_extended = (
-                    f"RDS Instance {db_instance.id} is not Publicly Accessible."
+                    f"RDS Instance {db_instance.id} is not publicly accessible."
                 )
             else:
                 report.status = "FAIL"
                 report.status_extended = (
-                    f"RDS Instance {db_instance.id} is set as Publicly Accessible."
+                    f"RDS Instance {db_instance.id} is set as publicly accessible."
                 )
-
+                # Check if restricted in DB Instance Security Group
+                if db_instance.security_groups:
+                    for security_group in ec2_client.security_groups:
```

Comment:
What happens if there are some security groups, some of them opened and others closed?

Reply:
If one is closed, it is prioritized.